tracer/src/Datadog.Trace/AspNet/TracingHttpModule.cs

```diff
@@ -177,6 +191,11 @@ private void OnBeginRequest(object sender, EventArgs eventArgs)
                     tracer.TracerManager.SpanContextPropagator.Inject(injectedContext, requestHeaders.Wrap());
                 }
 
+                if (inferredProxyScope is not null)
+                {
+                    httpContext.Items[_httpContextScopeKey + ".proxy"] = inferredProxyScope;
```

Comment:
We should save `_httpContextScopeKey + ".proxy"` into a field to avoid the concatenation and heap allocation on every http request. We can probably set it in [the ctor](https://github.com/DataDog/dd-trace-dotnet/blob/a134b4936bbcc2e029db6d496086412671ac6bc1/tracer/src/Datadog.Trace/AspNet/TracingHttpModule.cs#L65) like `_httpContextScopeKey`:

```diff
  _requestOperationName = operationName;
  _httpContextScopeKey = string.Concat("__Datadog.Trace.AspNet.TracingHttpModule-", _requestOperationName);
+ _httpContextScopeKeyProxy = string.Concat(_httpContextScopeKey, ".proxy");
```

Reply:
> to avoid the concatenation and heap allocation on every http request

_three_ string allocations, since we use `_httpContextScopeKey + ".proxy"` multiple times

Reply:
👍 

Thanks updated